test/integration/render-tests/icon-text-fit/both-collision-variable-anchor/style.json

```diff
@@ -0,0 +1,69 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64
+    }
+  },
+  "sources": {
+    "geojson": {
+      "type": "geojson",
+      "data": {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "geometry": {
+              "type": "Point",
+              "coordinates": [ 0, -8 ]
+            }
+          },
+          {
+            "type": "Feature",
+            "geometry": {
+              "type": "Point",
+              "coordinates": [ 0, -8 ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "sprite": "local://sprites/sprite",
+  "glyphs": "local://glyphs/{fontstack}/{range}.pbf",
+  "layers": [
+    {
+      "id": "symbol",
+      "type": "symbol",
+      "source": "geojson",
+      "layout": {
+        "text-field": "ABC",
+        "text-anchor": "top",
+        "text-offset": [0, 0.5],
+        "text-variable-anchor": ["left", "right"],
```

Comment:
I believe this test is showing a bug. The expected image is showing both left and right placements when the radial offset is provided. The correct expectation would be with only one of left or right rendered.

Reply:
I think it is correct, 2 features with same coordinates that are rendered by 2 layers. Only 1 feature from layer `"symbol_with_offset"` is rendered, as the second is hidden due to the collision. Layer with variable anchors is able to render both by trying placing these with radial offset.

Test verifies that icon collision box from `"symbol_with_offset"` layer is correctly shifted with a text box and does not collide with labels from `"symbol"` layer.